nemo_rl/data/datasets/response_datasets/__init__.py

```diff
@@ -61,6 +61,8 @@ def load_response_dataset(data_config, seed: int = 42):
             data_config["chat_key"],
             data_config["system_key"],
             data_config["system_prompt"],
+            data_config.get("tool_key", "tools"),
+            data_config.get("use_preserving_dataset", False),
```

Comment:
nit: i would prefer this appear in the sft.yaml so that users know what these are set to and that they can even set it
```suggestion
            data_config["tool_key"],
            data_config["use_preserving_dataset"],
```